e2e/tests/addOrRemoveLegalCounsel_test.js

```diff
@@ -86,11 +91,11 @@ Scenario('Legal counsel to be removed when respondent representative is removed
   assertLegalCounsellorForParties(I, ['Child 1']);
 });
 
-Scenario('Legal counsel to be removed when child representative is updated @flaky', async ({ I, caseViewPage, enterChildrenEventPage }) => {
+Scenario('Legal counsel to be removed when child representative is updated @fix-flaky-Feb', async ({ I, caseViewPage, enterChildrenEventPage }) => {
   checkLegalCounselWasAdded();
   await I.navigateToCaseDetailsAs(config.hmctsAdminUser, caseId);
   await addChildMainRepresentative(I, caseViewPage, enterChildrenEventPage, solicitor2);
-  I.seeEventSubmissionConfirmation(config.administrationActions.amendChildren);
+  // I.seeEventSubmissionConfirmation(config.administrationActions.amendChildren);
```

Comment:
Can we leave this in (as we've changed this function above)?

Reply:
Yeah, you are right. Thanks!